client/components/Project/ProjectDashboard.vue

```diff
@@ -1,11 +1,80 @@
 <template>
-  <section>
-    ProjectDashboard
-  </section>
+  <main>
+    <h1>Group Projects</h1>
+    <div class="project-filters">
+      <p
+        v-for="(value, label) in FILTER_TO_VALUE"
+        @click="activeFilter = value"
+        class="filter"
+        :class="{active: activeFilter === value}"
+      >
+        {{ label }}
+      </p>
+    </div>
+    <section>
+      <ProjectComponent
+        v-for="project in filteredProjects"
+        :key="project.id"
+        :project="project"
+      />
+    </section>
+  </main>
 </template>
 
 <script>
+import ProjectComponent from '@/components/Project/ProjectComponent.vue';
+
+const FILTER_TO_VALUE = {
+  'All': null,
+  'Active': true,
+  'Completed': false,
+}
+
 export default {
-  name: 'ProjectDashboard'
+  name: 'ProjectDashboard',
+  components: {ProjectComponent},
+  data() {
+    return {
+      // filter on the status of the project. null means all projects are shown
+      activeFilter: null,
+    };
+  },
+  beforeMount() {
+    this.$store.commit('refreshProjects');
+  },
+  created() {
+    this.FILTER_TO_VALUE = FILTER_TO_VALUE;
+  },
+  computed: {
+    filteredProjects() {
+      return this.$store.state.projects.filter(project => {
+        if (this.activeFilter === null) {
+          return true;
+        }
+        if (this.activeFilter && project.active) {
+          return true;
+        }
+        if (!this.activeFilter && !project.active) {
```

Comment:
is this if block necessary? or if its to catch a false value, can we catch that in first if statement?